WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/BadgeView.swift

```diff
@@ -89,10 +126,12 @@ private extension BadgeView {
     enum Layout {
         static let horizontalPadding: CGFloat = 6
         static let verticalPadding: CGFloat = 4
-        static let cornerRadius: CGFloat = 8
+        static let borderLineWidth: CGFloat = 1
+        static var cornerRadius: CGFloat { 8 }
```

Comment:
How come this is not like this?
`cornerRadius: CGFloat = 8`

Reply:
Oh that's because originally I attempted to make the change using generics, and there are limitations when using static stored properties that the compiler is ok with if we made them computed instead. Since we no longer follow this approach, I switched back to `let ` on https://github.com/woocommerce/woocommerce-ios/pull/12534/commits/722d304dc680ed4eb610f65f26f6179c82b506dd